doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -463,26 +506,29 @@ To configure the Hash algorithms, set one or more of the following Kconfig optio
-+-----------------------+---------------------------------------------------------------+---------------------------------------------------------------+
-| SHA-512               | Not supported                                                 | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_SHA_512_OBERON` |
-+-----------------------+---------------------------------------------------------------+---------------------------------------------------------------+
-| MD5 (weak)            | Not supported                                                 | Not supported                                                 |
-+-----------------------+---------------------------------------------------------------+---------------------------------------------------------------+
-| RIPEMD-160            | Not supported                                                 | Not supported                                                 |
-+-----------------------+---------------------------------------------------------------+---------------------------------------------------------------+
-
+You can use the following table to check the Hash algorithm support of each driver:
+
++-----------------------+----------------------------+---------------------------+
+| Hash algorithm        |  nrf_cc3xx driver support  | nrf_oberon driver support |
++=======================+============================+===========================+
+| SHA-1 (weak)          | Supported                  | Supported                 |
++-----------------------+----------------------------+---------------------------+
+| SHA-224               | Supported                  | Supported                 |
++-----------------------+----------------------------+---------------------------+
+| SHA-256               | Supported                  | Supported                 |
++-----------------------+----------------------------+---------------------------+
+| SHA-384               | Not supported              | Supported                 |
++-----------------------+----------------------------+---------------------------+
+| SHA-512               | Not supported              | Supported                 |
++-----------------------+----------------------------+---------------------------+
+| MD5 (weak)            | Not supported              | Not supported             |
++-----------------------+----------------------------+---------------------------+
+| RIPEMD160             | Not supported              | Not supported             |
++-----------------------+----------------------------+---------------------------+
+
+The option :kconfig:option:`CONFIG_PSA_USE_CC3XX_HASH_DRIVER` enables the driver :ref:`nrf_security_drivers_cc3xx` for all the supported algorithms.
+
+The driver configuration of the :ref:`nrf_security_drivers_oberon` driver is automatically generated based on the user enabled algorithms in  `HASH configurations`_.
```

Comment:
```suggestion
The configuration of the :ref:`nrf_security_drivers_oberon` driver is automatically generated based on the user enabled algorithms in  `HASH configurations`_.
```

Reply:
Search-replaced "The driver configuration" with "The configuration" in the file